content/docs/Configuration/examples.md

```diff
@@ -229,6 +229,27 @@ jobs:
   ```
 {{< /details >}}
 
+{{< details title="Tag cloud resources in Testing Farm" >}}
+Tag cloud resources in Testing Farm to a specific team or project.
```

Comment:
```suggestion
Tag cloud resources in Testing Farm to a specific Red Hat team or a project. If you are not a Red Hat employee, this section is not relevant for you.
```
So this section doesn't confuse upstream projects. It should be up to SSTs and us (Packit & TF) to make sure cloud costs are correctly monitored.

Reply:
Just to check, is it important for the public TF ranch as well?